internal/testrunner/runners/system/tester.go

```diff
@@ -122,6 +122,9 @@ var (
 			},
 		},
 	}
+	dataStreamOverrides = map[string]bool{
+		"amazon_security_lake": true,
```

Comment:
We don't add exceptions to specific packages in elastic-package. We should have generic mechanism so this is available for any package that supports it.

On this specific case, we have code to support routing rules, as well as custom datasets. If this is not enough for this package we should check why.

Reply:
Support for reroute was added in https://github.com/elastic/elastic-package/pull/1372 and https://github.com/elastic/elastic-package/pull/1391.

Reply:
The examples and the test package in https://github.com/elastic/elastic-package/pull/1372 are integration packages.